### ISSUE TITLE
pretrained flownetc

### ISSUE THREAD

**Issue by lraxue** [ClementPinard/FlowNetPytorch#45]
Are you going to release pretrained FlownetC model on FlyingChairs? Thank you.

**Comment by ClementPinard** [comment]
I just updated the README, you can now download it, the EPE is down to 1.766 !

**Comment by jeffbaena** [comment]
Dear Clement,
 thanks for your implementation. I have tested the network FlowNetC, with weights "flownetc_EPE1.766.tar" in sintel training clean (1041 frame pairs). The result is much better than the original: 
- yours EPE 2.26 px
-original EPE 4.31 px
 I would like to obtain the exact performance of FlowNetC.
 Do you have the weights for that? Alternatively, do you have information (or a script) with the exact training schedule and data augmentation?

Congratulations on your great work!

Bests,
Stefano

**Comment by ClementPinard** [comment]
For my training parameters, see README, especially [here](https://github.com/ClementPinard/FlowNetPytorch#training-results)

For original author's training parameters, the different hyperparameters (solver parameters, data augmentation etc) can be found inside their caffe archive. I tried to get as close as possible for the solver parameters (learning rate, betas, epochs etc), but I applied my own data augmentation without worrying too much about consistency with original paper, mostly because the current implentation made the results better. :stuck_out_tongue:
